functions/src/utils/camel-to-kebab.js

```diff
@@ -0,0 +1,23 @@
+const noneFirstUpperCaseRegex = /[A-Za-z]([A-Z])/g;
+const upperCaseWithUnderscoreAheadRegex = /[A-Za-z]_([A-Z])/g;
+
+/**
+ * Convert CamelStyle to kebab-style
+ *
+ * @param {string} value
+ * @returns {string}
+ */
+module.exports = (value) => {
+  const camelRes = value.replace(
+    noneFirstUpperCaseRegex,
+    s => `${s[0]}-${s[1].toLowerCase()}`
```

Comment:
I don't have a definite answer here, because I'm trying to reduce the verbosity of util functions, especially because they are pure functions and could be called multiple times what could flood the logs. But in more complex or less clean cases it would be very helpful. 
So I'm left this question open and we can always add extra logging, but if you feel that we should add logger here - I could add logging.

Reply:
Hadn't thought about how much noise that would add to the logs! I think you're right - logging should only be necessary for the more complicated stuff.